hack/release/verify-emoji.sh

```diff
@@ -25,5 +25,10 @@ base_link=$(jq -r '.pull_request.url' < ${GITHUB_EVENT_PATH})
 head_commit=$(curl -H "Authorization: Bearer ${GITHUB_TOKEN}" -H 'Accept: application/vnd.github.antiope-preview+json' -q ${base_link} | jq -r '.head.sha')
 echo "head commit is ${head_commit}"
 
-curl https://api.github.com/repos/${GITHUB_REPOSITORY}/check-runs -XPOST -H "Authorization: Bearer ${GITHUB_TOKEN}" -H 'Accept: application/vnd.github.antiope-preview+json' -H 'Content-Type: application/json' -q --data-raw '{"name": "Verify Emoji", "head_sha": "'${head_commit}'", "conclusion": "'${conclusion}'", "status": "completed", "completed_at": "'$(date -Iseconds)'", "output": {"title": "Verify Emoji", "summary": "'"${summary}"'"}}'
+echo ${summary}
```

Comment:
Maybe it's just me, but I'm not actually seeing the output of this in the result.  Let me try manually re-running.

Reply:
ah, it just shows up way at the bottom.  Need to scroll.